pages/api/admin/setup-links/index.ts

```diff
@@ -73,7 +78,7 @@ const handleGET = async (req: NextApiRequest, res: NextApiResponse) => {
 
   // Get a setup link by service
   if (service) {
-    const setupLink = await setupLinkController.getByService(service);
+    const setupLink = await setupLinkController.getByService(service, +(offset || 0), +(limit || 0));
```

Comment:
Can we rename `setupLink` to `setupLinks`? because `getByService` method returns an array.